whisper/src/resize.rs

```diff
@@ -1,19 +1,18 @@
 use crate::aggregation::AggregationMethod;
 use crate::builder::WhisperBuilder;
-use crate::error;
+use crate::error::Error;
 use crate::interval::Interval;
 
 use crate::point::Point;
 use crate::retention::Retention;
 use crate::WhisperFile;
 
-use failure::Error;
 use std::fs::{remove_file, rename};
 use std::io;
 use std::path::{Path, PathBuf};
 use std::process::exit;
 
-fn migrate_aggregate(path_src: &Path, path_dst: &Path, now: u32) -> Result<(), Error> {
+fn migrate_aggregate(path_src: &Path, path_dst: &Path, now: u32) -> io::Result<()> {
```

Comment:
Result type is inconsistent with other functions in the same module.

Reply:
because it returns only `io::Error`, while others mixed?
Or do you want `Result<(), io::Error>`

Reply:
These functions are non-public. They can be kept as is.